tests/drivers/i2c/i2c_api/src/test_i2c.c

```diff
@@ -28,6 +28,16 @@
 
 uint32_t i2c_cfg = I2C_SPEED_SET(I2C_SPEED_STANDARD) | I2C_MODE_CONTROLLER;
 
+#define HMC_ADDR (0x1E)
```

Comment:
Please define in Kconfig and use `extra_configs` in testcase.yaml.

Reply:
@soburi , It's updated please have a look